test/alternator/test_lsi.py

```diff
@@ -538,3 +539,35 @@ def test_lsi_and_gsi_same_name(dynamodb):
                 }
             ])
         table.delete()
+
+# Test that the LSI table can be addressed in Scylla's REST API (obviously,
+# since this test is for the REST API, it is Scylla-only and can't be run on
+# DynamoDB).
```

Comment:
maybe add `scylla_only` fixture? (I even suspect that rest_api fixture could depend on it)

Reply:
Indeed, "rest_api" already depends on scylla-only, and it's self-explanatory that it does (the "REST API" is a feature that only exists on Scylla).
Moreover, the "rest_api" needs **more** than just Scylla. We need to have Scylla that runs the REST API and its REST API port is open. This is not necessarily the case if Scylla is remote.

Reply:
"REST API"  - since dynamodb is kinda REST it may be more or less clear depends who reads :)

it doesn't depend on scylla_api fixture, but I see now it's doing pretty much the same check with different message so it's fine

Reply:
Yes, the phrase "rest api" is a Scylla jargon that perhaps we should have named differently, but it stuck.
The comments on this fixture explain that it is about "Scylla-specific REST API requests"